CMakeLists.txt

```diff
@@ -130,9 +130,28 @@ endif()
 
 set(DEFAULT_GPU_ARCH "" CACHE STRING "Optional: Default GPU architecture to compile for when targeting GPUs (e.g.: sm_60 or gfx900)")
 set(ROCM_LIBS "-L${ROCM_PATH}/lib -L${ROCM_PATH}/hip/lib -lamdhip64" CACHE STRING "Necessary libraries for ROCm")
-set(ROCM_LINK_LINE "-rpath ${ROCM_PATH}/lib -rpath ${ROCM_PATH}/hip/lib ${ROCM_LIBS}")
+if(NOT ROCM_LINK_LINE)
+  set(ROCM_LINK_LINE "-rpath ${ROCM_PATH}/lib -rpath ${ROCM_PATH}/hip/lib ${ROCM_LIBS}")
```

Comment:
Assuming `$ROCM_PATH` is always set and points to the detected ROCm installation as suggested above, also use `$HIPSYCL_ROCM_PATH` here?

PS: It seems newer clang versions have `--rocm-path` which seems a bit more clever than `--hip-device-lib-path`

Reply:
I use `$HIPSYCL_ROCM_PATH` should I also use the `--rocm-path` now? It depends on which version of clang we primarily support.

Reply:
I don't think we can roll out `--rocm-path` already. The clang-11 from apt.llvm.org that is used in our CI still seems to be happy without `--rocm-path`, so at the moment it seems only the clang 11 from Arch Linux seems to have it..